app/client/cli/debug.go

```diff
@@ -25,8 +25,14 @@ const (
 	PromptTogglePacemakerMode    string = "TogglePacemakerMode"
 	PromptShowLatestBlockInStore string = "ShowLatestBlockInStore"
 
-	defaultConfigPath  = "build/config/config1.json"
+	defaultConfigPath = "build/config/config1.json"
+	// Genesis file to be used from within a docker container to connect to other containers
 	defaultGenesisPath = "build/config/genesis.json"
+	// HACK: Genesis file to be used from the host machine to connect to docker containers.
+	// Note that `genesis.json` is a copy-pasta of `genesis_localhost.json` with the only
+	// difference being that `node{X}.consensus:8080` is replaced with `localhost:808{X}` because
+	// container names cannot be resolved by Docker's DNS from the host.
+	hostGenesisPath = "build/config/genesis_localhost.json"
```

Comment:
In new LocalNet we do not expose these ports. We can, even with scaling actors dynamically, but I first want to understand what is the benefit of executing this binary directly versus in a container (what `make client_connect` already does)?

Reply:
Similarly, this can be switched to an environment variable—happy to help and adjust the `p1` bash function, as we are going to hit a few git conflicts here anyway.

https://github.com/pokt-network/pocket/blob/1261079d7bfe62b747809ad52470c3720888423f/app/client/cli/debug.go#L42

Reply:
The `p1` binary has a `debug` option that is fundamentally broken right now. This was an attempt at fixing it

<img width="960" alt="Screenshot 2023-02-02 at 8 36 18 PM" src="https://user-images.githubusercontent.com/1892194/216513513-76abc5da-c4dd-447b-af92-309f8dda0592.png">